azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/CallingViewModel.kt

```diff
@@ -148,6 +148,7 @@ internal class CallingViewModel(
 
         audioDeviceListViewModel.update(
             state.localParticipantState.audioState.device,
+            state.localParticipantState.audioState.bluetoothDeviceName,
```

Comment:
Should we consider to use a bluetooth dedicated class in the audioState?

Reply:
A dedicated Bluetooth class has been added, and the ViewModels for SetupControlBarView and AudioDeviceListView have been updated to bring in this parent class (instead of each of its fields independently).